main/social/vcard_export.php

```diff
@@ -24,6 +23,14 @@
 // Return User Info to vCard Export
 $userInfo = api_get_user_info($userId, true, false, true);
 
+/* Get the relationship between current user and vcard user */
+$currentUserId = api_get_user_id();
+$hasRelation = (int) (SocialManager::get_relation_between_contacts($currentUserId, $userId));
```

Comment:
Don't add "(int)" better make sure that SocialManager::get_relation_between_contacts returns an int value doing the cast inside the function.

Reply:
indeed, the function get_relation_between_contacts returns the id of the record or the USER_UNKNOWN constant, for security I set the int in the case that row ['id'] returns a string

Reply:
Yes, exactly.